core/src/main/java/org/jdbi/v3/argument/SqlTypeArgument.java

```diff
@@ -11,19 +11,20 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jdbi.v3;
+package org.jdbi.v3.argument;
 
 import java.sql.PreparedStatement;
 import java.sql.SQLException;
 
+import org.jdbi.v3.StatementContext;
 import org.jdbi.v3.tweak.Argument;
 
-class SqlTypeArgument implements Argument
+public class SqlTypeArgument implements Argument
```

Comment:
Seems like this class and `ObjectArgument` are covering the same ground. Am I missing something?